pages/blog/a_straight_guide_to_apis_and_architecture_concepts.md

```diff
@@ -6,7 +6,7 @@ type: Communication
 tags:
   - APIs
   - Architectures
-cover: /img/posts/a_straight_guide_to_APIs_and_architecture_concepts/cover.webp
+cover: /img/posts/a_straight_guide_to_apis_and_architecture_concepts/cover.webp
```

Comment:
This change has been made to resolve the error of the image not rendering in the blog in the development environment (i.e. while running in local system).